.env.example

```
@@ -11,9 +11,11 @@ MIDAZ_ENVIRONMENT=local
 MIDAZ_USER_AGENT=Midaz-Go-SDK/1.0.0  # User agent string for API requests
 
 # API URLs (defaults based on environment if not specified)
-MIDAZ_BASE_URL=http://localhost
-MIDAZ_ONBOARDING_URL=http://localhost:3000/v1
-MIDAZ_TRANSACTION_URL=http://localhost:3001/v1
+# The SDK resolves MIDAZ_BASE_URL to the unified Ledger API base under /v1.
+MIDAZ_BASE_URL=http://localhost:3002
+MIDAZ_ONBOARDING_URL=http://localhost:3002/v1
+MIDAZ_TRANSACTION_URL=http://localhost:3002/v1
+MIDAZ_CRM_URL=http://localhost:4003/v1
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MIDAZ_CRM_URL key should go before the MIDAZ_ONBOARDING_URL key

(UnorderedKey)